opamp-client/src/http/client.rs

```diff
@@ -96,6 +97,11 @@ where
     // A sync method that handles the process of sending an AgentToServer message,
     // receives a ServerToAgent message, and analyzes the resulting message to decide
     // whether to resend or remain synced.
+    #[instrument(
+        name = "opamp::send_process",
+        fields(instance_uid = self.instance_uid.to_string()),
```

Comment:
Silly question as I never actually tried. Does this work?

```suggestion
        fields(instance_uid = %self.instance_uid),
```